eventq_aws/lib/eventq_aws/aws_queue_worker.rb

```diff
@@ -115,6 +116,7 @@ def thread_process_iteration(client, manager, queue, block)
 
           #check that a message was received
           if response.messages.length > 0
+            tag_processing_thread
```

Comment:
No, because that means the id is fixed per thread.  The idea is that a new id is generated for each round of work it does.  That way we can track per message processing all the log messages that applied to it.